README.md

```diff
@@ -2,9 +2,9 @@
 [![Build Status](https://travis-ci.org/ustwo/baseview-swift.svg?branch=master)](https://travis-ci.org/ustwo/baseview-swift)
 [![codecov.io](https://codecov.io/github/ustwo/baseview-swift/coverage.svg?branch=master)](https://codecov.io/github/ustwo/baseview-swift?branch=master)
 
-# BaseView
+# BaseViewSwift
 
-The BaseView framework provides an organizational tool for writing custom views using `UIKit`. This framework is written using Swift 2.1. It has been developed and used by iOS developers at [ustwo](ustwo.com).
+The BaseViewSwift framework provides an organizational tool for writing custom views using `UIKit`. This framework is written using Swift 2.1. It has been developed and used by iOS developers at [ustwo](ustwo.com).
```

Comment:
The suffix is indeed to keep it separate from the Obj-C version. That way they could both be packaged and not conflict in the dependency manager directory.